cypari2/pari_instance.pyx

```diff
@@ -616,14 +589,6 @@ cdef class Pari(Pari_auto):
         Print the internal PARI variables ``top`` (top of stack), ``avma``
         (available memory address, think of this as the stack pointer),
         ``bot`` (bottom of stack).
-
-        EXAMPLES::
-
-            sage: pari.debugstack()  # random
```

Comment:
No without a hook to doctest. This is what they did in cypari and what we might want to do... (they also have a hook for 32 vs 64 bits doctests)